cli/cfncluster/cfnconfig.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, args):
 
         # Determine the EC2 region to used used or default to us-east-1
         # Order is 1) CLI arg 2) AWS_DEFAULT_REGION env 3) Config file 4) us-east-1
-        if args.region:
+        if hasattr(args, "region") and args.region:
```

Comment:
Pretty small style thing I guess but you used single quotes previously and double quotes here

Reply:
fixed